src/data/faq_de.json

```diff
@@ -470,7 +470,8 @@
                         "anchor": "test_in_other_country",
                         "active": true,
                         "textblock": [
-                            "Nein, das ist leider nicht möglich. Jedes positive Test-Ergebnis, das in die App geladen wird, muss zuvor von Laboren überprüft worden sein. Diese stellen sicher, dass kein Missbrauch stattfindet und keine falsche Ergebnisse eingetragen werden. Dazu sind zur Zeit nur Labore in Deutschland angeschlossen. Sie können also kein positives Testergebnis aus anderen Ländern, zum Beispiel aus den Niederlanden, eingeben."
+                            "Sie können ein positives Coronavirus-Testergebnis nur in der deutschen Corona-Warn-App eingeben, wenn der Test in Deutschland durchgeführt wurde. Ein positives Testergebnis, das Sie von einem Auslandslabor erhalten haben, müssen Sie in der jeweiligen App des Landes eingeben. Dies stellt sicher, dass kein Missbrauch stattfindet und keine falschen Ergebnisse eingetragen werden. Für Mitgliedsländer der EU stellt die EU-Kommission <a href='https://ec.europa.eu/info/live-work-travel-eu/health/coronavirus-response/travel-during-coronavirus-pandemic/mobile-contact-tracing-apps-eu-member-states_de'>eine Liste allen europäischen Apps</a> zur Verfügung.",
```

Comment:
Since the wording should be as close as possible, I woul recommend changing the third sentence to 

>  Dies stellt sicher, dass Ergebnisse nicht missbräuchlich eingetragen werden können.

Also, "Mitgliedsländer" should be changed to "Mitgliedsstaaten".

Reply:
Changed to "Dies stellt sicher, dass Ergebnisse nicht missbräuchlich eingetragen werden können." which I also prefer.

"Für Mitgliedsländer der EU" changed to "Den EU-Mitgliedstaaten".
See https://europa.eu/european-union/about-eu/countries_de for spelling of Mitgliedstaaten (only one "s").